### PR TITLE
Integration of the first version of the indicator detection

### PR DESCRIPTION
**Pull request by MichaelRoeder**

### CODE REVIEW

File: org/diceresearch/nebula/indicators/main.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        if indicator_results is not False
+    } 
+'''
+
+
+def _run_topical_checks():
+    pass
+
+
+def run_indicator_check(json_input: Any):
```
Comment (MichaelRoeder) [inline org/diceresearch/nebula/indicators/main.py:72]: @umairq  This needs to be called for the integration.

**Review by MichaelRoeder — COMMENTED** [review]
Looks good so far. However, the integration is missing. I left a comment at the function that needs to be called.

@umairq Can you please take care of that?

**Comment by AlyMoh** [comment]
@MichaelRoeder we are moving the development of the indicator detection component to a separate git repository hosted on our self-managed GitLab server. I was considering replacing the current feature indicator structure with a git submodule in order to keep both repositories separate yet allow you to access the latest working version of the component. Would that be suitable on your end?

**Comment by MichaelRoeder** [comment]
That is fine by me. I would like to merge the current state within the next week to ensure that we have a first version integrated for December.

Is your gitlab public? Otherwise, a git sub-module won't work as expected, right?

**Comment by AlyMoh** [comment]
~Our GitLab repository is private, yet technically a git submodule setup is possible.~

~Since I have access to both repositories (`dice-group/NEBULA` and ours) I can update the submodule within `dice-group/NEBULA` whenever there's an update to the feature detection component. From your point of view, it'd be just like any other folder which can be accessed within the GitHub repository. I just took it to be a somewhat cleaner approach than copying and pasting files and folders over (otherwise the git log would display hundreds of changes which is from your point of view probably unnecessary) and it ensures that any files and folders needed for our component to work are not overlooked.~

~Of course we can carry out this change only once we have a first integrated version by December to avoid any unnecessary complications.~

Scratch that. My understanding of git submodules was flawed. You'd be able to see the URL to the private repository on GitLab, but cloning and working with the contents on your end would not be possible.

(There seems to be a way of achieving this through GitHub Actions, but honestly since we intend on migrating to a fully decoupled architecture, it makes little sense to go through with this suggestion.)

I guess the best course of action in the event of updates would be to to manually push them into `dice-group/NEBULA`. Should you have any questions or concerns regarding integration of the current version, feel free to reach out at any time.

**Pull request merged**